nevergrad/functions/control/test_mujoco.py

```diff
@@ -3,24 +3,49 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import pytest
 import numpy as np
 from .core import BaseFunction
 
+
 class MountainCarContinuous(BaseFunction):
     env_name = "MountainCarContinuous-v0"
-    policy_dim= (1, 2)
+    policy_dim = (2, 1)
     state_mean = [0, 0]
     state_std = [1, 1]
 
+
 def test_gym() -> None:
-    func = MountainCarContinuous(num_rollouts=5)
+    for intermediate_layer_dim in [None, (10,), (5, 5)]:
+        for noise_level in [0.0, 0.9]:
+            for deterministic_sim in [True, False]:
+                for states_normalization in [True, False]:
```

Comment:
this tests now takes 10s which is a bit long for a test :s . Sometimes there is nothing we can do about it (unfortunately :s), but here can we either or both:
- use fewer rollout
- not test the whole grid of parameters combinations, picking a few test cases should be enough to test each elements if they are uncoupled enough

Also, please use [pytest.mark.parametrize(https://docs.pytest.org/en/stable/parametrize.html) for providing the test cases, since this splits the test and allows to figure out more easily which cases break (+ it will be more readable, with fewer indented for loops)